systemtest/src/test/java/io/strimzi/systemtest/operators/NamespaceDeletionRecoveryST.java

```diff
@@ -196,27 +176,7 @@ private void prepareEnvironmentForRecovery(TestStorage testStorage) {
             .createInstallation()
             .runInstallation();
 
-        resourceManager.createResourceWithWait(
-            NodePoolsConverter.convertNodePoolsIfNeeded(
-                KafkaNodePoolTemplates.brokerPoolPersistentStorage(testStorage.getNamespaceName(), testStorage.getBrokerPoolName(), testStorage.getClusterName(), 3)
-                    .editSpec()
-                        .withNewPersistentClaimStorage()
-                            .withSize("1Gi")
-                            .withStorageClass(storageClassName)
-                        .endPersistentClaimStorage()
-                    .endSpec()
-                    .build(),
-                KafkaNodePoolTemplates.controllerPoolPersistentStorage(testStorage.getNamespaceName(), testStorage.getControllerPoolName(), testStorage.getClusterName(), 3)
-                    .editSpec()
-                        .withNewPersistentClaimStorage()
-                            .withSize("1Gi")
-                            .withStorageClass(storageClassName)
-                        .endPersistentClaimStorage()
-                    .endSpec()
-                    .build()
-            )
-        );
-        resourceManager.createResourceWithWait(KafkaTemplates.kafkaPersistent(testStorage.getNamespaceName(), testStorage.getClusterName(), 3, 3)
```

Comment:
this test `testTopicNotAvailable` actually fails with kraft on a step of recreating Kafka Cluster after namespace is deleted. All to be recreated pods get into crash-loop as they try to connect to incorrect cluster. 

For example: 

```
Exception in thread “main” java.lang.RuntimeException: Invalid cluster.id in: /var/lib/kafka/data/kafka-log1/meta.properties. Expected 28c5tB6tSlCjEHep6l3Jww, but read TRrzEInZRgCs_R9iNz2Gkw at org.apache.kafka.metadata.properties.MetaPropertiesEnsemble.verify(MetaPropertiesEnsemble.java:509) 
```

Reply:
This feels like a bug to me @scholzj @fvaleri ?

Reply:
@see-quick finally I was able to have a look at this.

> 1. deploy the Kafka cluster without Topic Operator - otherwise topics will be deleted

This is not needed anymore with the new Topic Operator, so it can be removed from the code.

---

To make this test work on my local Minikube I had to update the `StorageClass.VolumeBindingMode` from `WaitForFirstConsumer` to `Immediate`. Is there any Minikube configuration you are doing here? Is this documented somewhere?

---

I was able to reproduce the issue mentioned by @henryZrncik:

```sh
Exception in thread "main" java.lang.RuntimeException: Invalid cluster.id in: /var/lib/kafka/data/kafka-log0/meta.properties. Expected kQHv733NQIew9aw9uCXnDA, but read 2DLef4_8TqOVdBjzCORB1Q
```

I also think this is a bug, so I raised #10722. The attached fix should work with this test. Hope it helps.

Reply:
This is not a bug, it means that this was not set when recovering the Kafka CR.

Reply:
@im-konge we now have an updated procedure that may help with this test: https://strimzi.io/docs/operators/in-development/deploying#proc-cluster-recovery-volume-str

Reply:
I will update the test when I find some time thanks 💯